libs/types/src/index.ts

```diff
@@ -1,4 +1,5 @@
 /* istanbul ignore file */
+export * as Admin from './admin';
```

Comment:
There is VxAdmin specific code in `libs/ui`, yes. Since our reports share components (e.g. headers and tally tables), it made sense when adding VxAdmin specific reports to add them in `libs/ui`. That way, they are grouped with all the other reports and can benefit from the development tools (storybook) set up in `libs/ui`. 

In the future, I envision report components living in `libs/printing`, which `admin-backend` would depend on, so then it would make even more sense to define it elsewhere.

Open to other thoughts here. Splitting `Admin` types across `admin-backend` and `libs/types` is not very pretty, but it works.

Reply:
Ok, I can live with it for ease of using Storybook